zips/zip-0246.rst

```diff
@@ -513,21 +514,40 @@ auth_digest
 -----------
 A BLAKE2b-256 hash of the following values ::
 
-   A.1: transparent_scripts_digest (32-byte hash output)
-   A.2: sapling_auth_digest        (32-byte hash output)
-   A.3: orchard_auth_digest        (32-byte hash output)
-   A.4: issuance_auth_digest       (32-byte hash output)
+   A.1: transparent_scripts_digest       (32-byte hash output)
+   A.2: transparent_sighash_infos_digest (32-byte hash output)
+   A.3: sapling_auth_digest              (32-byte hash output)
+   A.4: orchard_auth_digest              (32-byte hash output)
+   A.5: issuance_auth_digest             (32-byte hash output)
 
 The personalization field of this hash remains the same as in ZIP 244.
 
+Note that while the structure of ``sapling_auth_digest`` remains the same as in ZIP 244, the field encodings of some of its components 
+(viz. ``spend_auth_sigs`` and ``binding_sig``) differ due to changes to their field encodings in ZIP 230 [#zip-0230-transaction-field-encodings]_.
+
+
+A.2: transparent_sighash_infos_digest
+`````````````````````````````````````
+
+A BLAKE2b-256 hash of the field encodings of all ``TransparentSighashInfo`` values belonging to the transaction.
+
+The personalization field of this hash is set to::
+
+    "ZTxAuthTrSHIHash"
+
+In the case that the transaction has no transparent inputs, ``transparent_sighash_infos_digest`` is::
+
+    BLAKE2b-256("ZTxAuthTrSHIHash", [])
+
```

Comment:
Isn't this redundant? It is implied by it being a hash of the concatenation of field encodings.
```suggestion
```

Reply:
I see what you mean, yes. I was following, for example, `T.2a`, `T.2b` and `A.1` in ZIP 244 when including these lines.
https://github.com/zcash/zips/blob/4cf2b44f3c82513c236323917f98c21bed9b2c17/zips/zip-0244.rst?plain=1#L215-L257
https://github.com/zcash/zips/blob/4cf2b44f3c82513c236323917f98c21bed9b2c17/zips/zip-0244.rst?plain=1#L688-L700

Would this comment apply there too? Though I also see that the fields being encoded are from Bitcoin, and not explicitly in the Transaction Format, unlike `TransparentSighashInfo`, so it is not as clearly implied.